src/ng-multiselect-dropdown/src/multiselect.component.ts

```diff
@@ -300,12 +300,15 @@ export class MultiSelectComponent implements ControlValueAccessor {
   }
 
   closeDropdown() {
-    this._settings.defaultOpen = false;
     // clear search text
     if (this._settings.clearSearchFilter) {
```

Comment:
Should this go inside the defaultOpen check also? This function probably shouldn't be doing anything at all if the dropdown wasn't already open.